packages/lab/src/slider/internal/SliderTrack.tsx

```diff
@@ -1,111 +1,112 @@
-
-  const [activeThumb, setActiveThumbState] =
-    useState<ActiveThumbIndex>(undefined);
-  const activeThumbRef = useRef<ActiveThumbIndex>(undefined);
-  const pointerDown = useRef(false);
-
-  const setActiveThumb = (index: ActiveThumbIndex) => {
-    setActiveThumbState(index);
-    activeThumbRef.current = index;
-  };
-
-  const handlePointerUp = () => {
-    setActiveThumb(undefined);
-    pointerDown.current = false;
-  };
-
-  const handlePointerDown = (event: React.PointerEvent<HTMLDivElement>) => {
-    if (event.button !== 0) return;
-    const { clientX } = event;
-    const newValue: number = getValue(trackRef, min, max, step, clientX);
-    const nearestIndex = getNearestIndex(value, newValue);
-    setValue(value, newValue, value.length > 1 ? nearestIndex : 0, onChange);
-    setActiveThumb(nearestIndex);
-    pointerDown.current = true;
-  };
-
-  const handlePointerMove = (event: PointerEvent) => {
-    if (!pointerDown.current || typeof activeThumbRef.current === "undefined")
-      return;
-    const thumb = activeThumbRef.current;
-    const { clientX } = event;
-    const rawValue: number = getValue(trackRef, min, max, step, clientX);
-    const newValue = preventOverlappingValues(
-      valueRef.current,
-      rawValue,
-      thumb,
+export const SliderTrack = forwardRef<HTMLDivElement, SliderTrackProps>(
+  function SliderTrack(
+    {
+      className: classNameProp,
+      children,
+      disabled,
+      handleMouseDown,
+      isDragging,
+      labelPosition,
+      markers,
+      max,
+      maxLabel,
+      min,
+      minLabel,
+      progressPercentage,
+      progressPercentageRange,
+      sliderRef,
+      ...rest
+    },
+    ref,
+  ) {
+    return (
+      <div
+        className={clsx(
+          withBaseName(),
+          {
+            [withBaseName("disabled")]: disabled,
+            [withBaseName("dragging")]: isDragging,
+            [withBaseName(`${labelPosition}Labels`)]: !markers,
+            [withBaseName("withMarkers")]: markers,
+          },
+          classNameProp,
+        )}
+        ref={ref}
+        {...rest}
+      >
+        {/* Min Label */}
+        {!markers && (
+          <Text as="label" className={withBaseName("minLabel")}>
+            {minLabel || min}
+          </Text>
+        )}
+        {/* Slider Track */}
+        <div
+          className={withBaseName("track")}
+          onMouseDown={handleMouseDown}
+          ref={sliderRef}
+          style={
+            {
+              "--slider-progressPercentage": `${progressPercentage}%`,
+              "--slider-progressPercentageStart": `${progressPercentageRange?.[0]}%`,
+              "--slider-progressPercentageEnd": `${progressPercentageRange?.[1]}%`,
+            } as React.CSSProperties
+          }
+        >
+          {children}
+        </div>
+        {/* Max label */}
+        {!markers && (
+          <Text as="label" className={withBaseName("maxLabel")}>
```

Comment:
```suggestion
          <Text styleAs="label" className={withBaseName("maxLabel")}>
```

Worth checking with @jake-costa but right now markers are `<label>` elements.

Reply:
@tayyabataimur Yeah, we shouldn't have those as labels and we should also provide aria-hidden to these as well.